tools/sklearn_tools/.shed.yml

```diff
@@ -0,0 +1,31 @@
+name: sklearn_tools
+owner: iuc
+description: Machine Learning tool suite from Scikit-learn
+
+long_description: |
+  Scikit-learn is an open source machine learning library written in python.
+  It provides different flavors of machine learning algorithms for classification,
+  regression, and clustering. It has designed to interoperate with python numerical
+  and scientific libraries Numpy and Scipy.
+  
+  The official repository of Scikit-learn is at https://github.com/scikit-learn/scikit-learn
+
+categories:
+  - Machine Learning
+  - Statistical Pattern Recognition
```

Comment:
These needs to correspond to the TS categories: https://testtoolshed.g2.bx.psu.edu/
`Statistics` would be fine.